plugin/clang.vim

```diff
@@ -765,9 +791,9 @@ func! s:ClangCompleteInit(force)
   let b:clang_options .= ' -I ' . shellescape(expand("%:p:h"))
 
   " add include directories if is enabled and not ow
+  let l:default_incs = s:DiscoverDefaultIncludeDirs(b:clang_options)
   if g:clang_include_sysheaders && ! l:is_ow
-    let l:incs = s:DiscoverIncludeDirs(g:clang_exec, b:clang_options)
-    for l:dir in l:incs
```

Comment:
Why call `s:DiscoverIncludeDirs ` outside of `if g:clang_include_sysheaders && ! l:is_ow` ?
I think should be in the `if` block.